numpy/core/src/multiarray/arrayfunction_override.c

```diff
@@ -208,16 +208,24 @@ call_array_function(PyObject* argument, PyObject* method,
 }
 
 
-/*
- * Implements the __array_function__ protocol for a function, as described in
- * in NEP-18. See numpy.core.overrides for a full docstring.
+/**
+ * Internal handler for the array-function dispatching. The helper returns
+ * either the result, or NotImplemented (as a borrowed reference).
```

Comment:
I think this (vs extra recounting) is fine as long as it's documented and internal. At some point, we should probably make this type of thing consistent, but I don't feel strongly about it.

Reply:
@seberg would you like to comment here given this was your idea? I don't feel strongly either way.

Reply:
No action needed, just expredsing agreement :)